control_panel_api/settings/base.py

```diff
@@ -1,6 +1,15 @@
 import os
 
-import boto3
+
+def is_enabled(value):
+    return str(value).lower() not in ('n', 'no', 'off', 'false', '0')
+
+
+# These are boolean flags to enable/disable features in the API
+ENABLED = {
+    'write_to_cluster':
```

Comment:
Shouldn't we have more granular control over which libraries are enabled?

Reply:
These are flags for features, not libraries. The feature set being enabled/disabled here is writing changes to the cluster. Having to set multiple flags for a single concept would be confusing and prone to errors.

Reply:
We might need to revisit if we have overlapping feature flags, but hopefully that would be rare as we'd remove old flags when they become unnecessary.

Reply:
👍